Dockerfile

```diff
@@ -25,10 +25,14 @@ RUN apt-get update && \
     apt-get install -y software-properties-common && \
     apt-add-repository -y ppa:deadsnakes/ppa && \
     apt-get update && apt-get upgrade -qy && \
-    apt-get install language-pack-en locales git python3.6 python3-pip libmysqlclient-dev libssl-dev python3-dev python3.8-dev python3.8-distutils -qy && \
-    pip3 install --upgrade pip setuptools && \
+    apt-get install language-pack-en locales git libmysqlclient-dev libssl-dev build-essential python3.8-dev python3.8-venv -qy && \
```

Comment:
@mraarif Would you be able to split this apt-get line into multiple lines like this:  https://github.com/edx/edx-cookiecutters/blob/master/cookiecutter-django-ida/%7B%7Bcookiecutter.repo_name%7D%7D/Dockerfile#L20-L29

https://openedx.atlassian.net/wiki/spaces/SRE/pages/1838350349/Dockerfile+Best+Practices

Reply:
You may also want to update the comment lines above this this RUN command to match your changes.

Reply:
sure